src/inputs/select.scss

```diff
@@ -64,3 +66,53 @@
     }
   }
 }
+
+@mixin iui-multi-select-tag-container {
+  position: absolute;
+  inset: 0 $iui-icons-default + $iui-l 0 $iui-sm;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  gap: $iui-xs;
+}
+
+@mixin iui-multi-select-tag {
+  @include iui-tag;
+  margin: 0;
+  display: inline-flex;
+  align-items: center;
+  height: 80%;
```

Comment:
What would you suggest? padding around is already pretty small.. If I increase height, there would be even worse :/

Reply:
Maybe we should use `basic` tags in small select? or would that be too confusing?

/cc @FlyersPh9

Reply:
I know we tried invisible tags, looked weird too :D
Basic would not have on hover, I guess.. But I had idea of just displaying text and separate with coma when got tired of tags...

Reply:
If we used basic tags I'd like it if the user hit the delete key that it would delete the entire tag instead of just 1 character from the tag.  1 benefit to using the basic tag over the other tag would be that we could display more tags because there is no wasted space for the X icon.

Reply:
> Have in mind, it is select, not comobobox

That seems weird to me.  If you can select multiple items, maybe you should need to use a combobox?

Reply:
hm.... Difference between select and combobox is that combobox has input where you can type, while select just shows list.
So idk.. I guess having multiselect in Select makes sense too :/